devel/semver-utils/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+compiler.cxx_standard  2011
+
+github.setup        emcrisostomo semver-utils 2.0.0
+revision 1
+github.tarball_from releases
+
+categories          devel
+license             GPL-3
+
+maintainers         gmail.com:enrico.m.crisostomo \
+                    openmaintainer
+description         C/C++ semantic version parser
+long_description    A C/C++ library and an utility to parse, compare and \
+                    manipulate versions numbers complying with Semantic \
+                    Versioning (https://semver.org).
+
+homepage            https://github.com/emcrisostomo/semver-utils
+
+checksums           rmd160  3f7a6c2108003e836fcfb4435caf9524a98b6369 \
+                    sha256  6228624b1fdd86965645449b9868bdceedfe1ee5397dca923d243dae45dd0265 \
+                    size    530426
+
+livecheck.url       ${github.homepage}/releases/latest
```

Comment:
this should not be needed; why dies the default value set by the github PortGroup not work?